File: java-unit-tests/src/main/java/com/example/second_module/Car.java
package com.example.second_module;

public class Car {
    int x;
    int y;

    public Car(int startPosX, int startPosY) {
        x = startPosX;
        y = startPosY;
    }

    public int[] getPosition() {
        int[] result = new int[2];
        result[0] = x;
        result[1] = y;
        return result;
    }

    public void move(int velocity_x, int velocity_y) {
        x += velocity_x;
        y += velocity_y;
    }
}

File: scala-unit-tests/src/main/scala/second_module/Car.scala
package second_module

// Пример тестирования поведения объектов класса
class Car(var x: Int, var y: Int):
  def get_position = (x, y)

  def move(velocity_x: Int, velocity_y: Int) =
    x += velocity_x
    y += velocity_y
